tests/hpc/mpi_master.pm

```diff
@@ -74,7 +74,12 @@ sub run ($self) {
 
     barrier_wait('MPI_SETUP_READY');
     record_info 'MPI_SETUP_READY', strftime("\%H:\%M:\%S", localtime);
-    assert_script_run("$mpi_compiler $exports_path{'bin'}/$mpi_c -o $exports_path{'bin'}/$mpi_bin") if $mpi_compiler;
+    if (get_var('HPC_LIB') eq 'papi') {
+        assert_script_run('module load gnu papi');
+        assert_script_run("$mpi_compiler $exports_path{'bin'}/$mpi_c -o $exports_path{'bin'}/$mpi_bin -I/usr/lib/hpc/papi/7.0.0/include/ -L/usr/lib/hpc/papi/7.0.0/lib64/ -lpapi") if $mpi_compiler;
```

Comment:
not sure about hardcoded version in path

Reply:
fixed it in updated PR - https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/16924